addons/account/wizard/account_inconsistant_bank.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+
+
+class AccountInconsistantBank(models.TransientModel):
+    _name = 'account.inconsistant.bank.wizard'
+    _description = ' Wizard for inconsistant bank account on partner when importing file'
+
```

Comment:
I don't think this is the right approach: a vendor bill could be pushed through a mail gateway and no one will ever receive feedback using this.